components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -277,6 +278,46 @@ export const Account = () => {
     })
   }, [selectedAccount, filteredRouteOptions])
 
+  const fungibleTokensSortedByValue = React.useMemo(() => {
+    if (!selectedAccount || isLoadingSpotPrices || isLoadingBalances) {
+      return fungibleTokens
+    }
+    return fungibleTokens.sort(function (a, b) {
```

Comment:
`.sort` mutates the array, we should be returning a new array to avoid the mutation

```suggestion
    return fungibleTokens.toSorted(function (a, b) {
```